libobs/obs-avc.c

```diff
@@ -202,6 +256,26 @@ size_t obs_parse_avc_header(uint8_t **header, const uint8_t *data, size_t size)
 	s_wb16(&s, (uint16_t)pps_size);
 	s_write(&s, pps, pps_size);
 
+	uint8_t profile_idc = sps[1];
+
+	/* Additional data required for high, high10, high422, high444 profiles.
+	 * See ISO/IEC 14496-15 Section 5.3.3.1.2 */
+	if (profile_idc == 100 || profile_idc == 110 || profile_idc == 122 ||
+	    profile_idc == 244) {
+		uint8_t chroma_format_idc, bit_depth_luma, bit_depth_chroma;
```

Comment:
```suggestion
		uint8_t chroma_format_idc;
		uint8_t bit_depth_luma;
		uint8_t bit_depth_chroma;
```

Reply:
We don't break up declarations of the same type unless they also contain assignments. And I'd prefer to not do that.

Reply:
I prefer distinct lines, but that's personal preference. Looks like the style guide permits this if there are no assignments.